plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol

```diff
@@ -89,7 +104,12 @@ library PaymentProcessInFlightExit {
         clearPiggybackOutputFlag(exit, token);
 
         if (allPiggybacksCleared(exit)) {
-            exit.bondOwner.transfer(exit.bondSize);
+            // solhint-disable-next-line avoid-call-value
+            (bool success, ) = exit.bondOwner.call.value(exit.bondSize)("");
+            // we do not want to block a queue if bond return is unsuccessful
+            if (!success) {
+                emit InFlightBondReturnFailed(exit.bondOwner, exit.bondSize);
```

Comment:
Does that mean that the funds get stuck in the contract forever?

Reply:
yes